.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish Github Pages
+
+on:
+  push:
+    branches: [master]
```

Comment:
it could be nice to have URL prefixes for different client library releases, e.g. `/latest/`, `/v0.15.1/`, etc. That way you can link to stable versions of the docs vs. latest